google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/StreamingSubscriberConnection.java

```diff
@@ -64,15 +54,27 @@
   private static final Logger logger =
       Logger.getLogger(StreamingSubscriberConnection.class.getName());
 
-  @InternalApi static final Duration DEFAULT_STREAM_ACK_DEADLINE = Duration.ofSeconds(60);
-  @InternalApi static final Duration MAX_STREAM_ACK_DEADLINE = Duration.ofSeconds(600);
-  @InternalApi static final Duration MIN_STREAM_ACK_DEADLINE = Duration.ofSeconds(10);
   private static final Duration INITIAL_CHANNEL_RECONNECT_BACKOFF = Duration.ofMillis(100);
   private static final Duration MAX_CHANNEL_RECONNECT_BACKOFF = Duration.ofSeconds(10);
+
+  private static final long INITIAL_ACK_OPERATIONS_RECONNECT_BACKOFF_MILLIS = 100;
+  private static final long MAX_ACK_OPERATIONS_RECONNECT_BACKOFF_MILLIS =
+      Duration.ofSeconds(10).toMillis();
   private static final int MAX_PER_REQUEST_CHANGES = 1000;
 
-  private final Duration streamAckDeadline;
-  private final SubscriberStub stub;
+  private final String PERMANENT_FAILURE_INVALID_ACK_ID_METADATA =
+      "PERMANENT_FAILURE_INVALID_ACK_ID";
+  private final String TRANSIENT_FAILURE_METADATA_PREFIX = "TRANSIENT_";
+
+  private Duration maxAckExtensionPeriod;
```

Comment:
We never use maxAckExtensionPeriod, minDurationPerAckExtension, minDurationPerAckExtensionDefaultUsed, maxDurationPerAckExtension, or maxDurationPerAckExtensionDefaultUsed in StreamingSubscriberConnection, other than in the constructor.

We don't need to store these values in StreamingSubscriberConnection, let's remove these added definitions.

Reply:
Removed